client/packages/system/src/InventoryAdjustmentReason/Components/InventoryAdjustmentReasonSearchInput.tsx

```diff
@@ -44,13 +46,14 @@ export const InventoryAdjustmentReasonSearchInput: FC<
   };
   const reasons = (data?.nodes ?? []).filter(reasonFilter);
 
-  const isRequired = reasons.length !== 0;
+  const reasonsAvailable = reasons.length !== 0;
+  const isRequired = reasonsAvailable && initialStocktake != true;
 
   return (
     <Box display="flex" flexDirection="row" width={120}>
       <Autocomplete
         autoFocus={autoFocus}
-        disabled={isDisabled || !isRequired}
+        disabled={isDisabled || !reasonsAvailable}
```

Comment:
Would just disable reasons if it's an initial stocktake
```suggestion
        disabled={isDisabled || !isRequired}
```

Reply:
Confirmed this is the intended behaviour - disable reasons on initial stocktakes :)